Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/azuredeploy.json

```diff
@@ -0,0 +1,351 @@
+                    "startDaysAgo": "-30",
+                    "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/cohesity/Azure-Sentinel/raw/CohesitySecurity.internal/Solutions/CohesitySecurity/Data%20Connectors/Helios2Sentinel/Package/IncidentProducer.zip"
+                }
+            }]
+        },
+        {
+            "type": "Microsoft.Web/sites",
+            "apiVersion": "2018-11-01",
+            "name": "[variables('ConsumerFunctionName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Storage/storageAccounts', tolower(variables('ProducerFunctionName')))]"
+            ],
+            "kind": "functionapp",
+            "identity": {
+                "type": "SystemAssigned"
+            },
+            "properties": {
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]",
+                "name": "[variables('ConsumerFunctionName')]",
+                "httpsOnly": true,
+                "clientAffinityEnabled": true,
+                "alwaysOn": true,
+                "reserved": true
+            },
+            "resources": [{
+                "type": "config",
+                "apiVersion": "2018-11-01",
+                "name": "appsettings",
+                "dependsOn": [
+                    "[concat('Microsoft.Web/sites/', variables('ConsumerFunctionName'))]"
+                ],
+                "properties": {
+                    "FUNCTIONS_EXTENSION_VERSION": "~4",
+                    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+                    "APPINSIGHTS_INSTRUMENTATIONKEY": "[reference(resourceId('Microsoft.insights/components', variables('ConsumerFunctionName')), '2015-05-01').InstrumentationKey]",
+                    "APPLICATIONINSIGHTS_CONNECTION_STRING": "[reference(resourceId('microsoft.insights/components', variables('ConsumerFunctionName')), '2015-05-01').ConnectionString]",
+                    "CohesityQueueName": "[variables('QueueName')]",
+                    "keyVaultName": "[variables('ProducerFunctionName')]",
+                    "resourceGroup": "[resourceGroup().Name]",
+                    "subscription": "[subscription().subscriptionId]",
+                    "TenantId": "[subscription().tenantId]",
+                    "AzureWebJobsStorage": "[concat('DefaultEndpointsProtocol=https;AccountName=', toLower(variables('ProducerFunctionName')),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', toLower(variables('ProducerFunctionName'))), '2019-06-01').keys[0].value, ';EndpointSuffix=',toLower(variables('StorageSuffix')))]",
+                    "Workspace": "[parameters('Workspace')]",
+                    "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/cohesity/Azure-Sentinel/raw/CohesitySecurity.internal/Solutions/CohesitySecurity/Data%20Connectors/Helios2Sentinel/Package/IncidentConsumer.zip"
```

Comment:
@yinghuang123 : please use below short url for **website_run_frompackage** path https://aka.ms/sentinel-Cohesity-functionapp-IncidentConsumer

Reply:
At this point, this shortened URL still points to our internal branch. Hope this will be changed to Azure master branch.

Reply:
@kishannerella, yes.